test/ring/middleware/test/json.clj

```diff
@@ -62,15 +62,81 @@
                    :body (string-input-stream "{\"foo\": \"bar\"")}]
       (is (= (handler request) malformed))))
 
-  (let [handler  (fn [_] {:status 200 :headers {} :body {:bigdecimals cheshire.parse/*use-bigdecimals?*}})]
-    (testing "don't overwrite bigdecimal binding"
+  (testing "don't overwrite bigdecimal binding"
```

Comment:
This change doesn't seem directly related to the PR. We should save formatting changes for their own commits/PRs so that all lines of the diff are relevant.